packages/ui-react/src/components/LanguageMenu/LanguageMenu.tsx

```diff
@@ -13,14 +18,54 @@ type Props = {
 };
 
 const LanguageMenu = ({ onClick, languages, currentLanguage }: Props) => {
+  const [languageMenuOpen, setLanguageMenuOpen] = useState(false);
+
+  const handleOnBlur = () => setLanguageMenuOpen(false);
+  const handleOnFocus = () => setLanguageMenuOpen(true);
+  const handleMenuToggle = () => setLanguageMenuOpen(!languageMenuOpen);
+  const handleLanguageSelect = (event: MouseEvent<HTMLElement>, code: string) => {
+    event.preventDefault();
+    if (onClick) {
+      onClick(code);
+    }
```

Comment:
Even more nice (short) ;-)

```suggestion
    onClick && onClick(code);
```

Reply:
Or `onClick?.(code);` 😜

Reply:
onClick && onClick(code). ->  Favor explicit checks? @royschut / @langemike

Reply:
An explicit check is not needed. Regarding choosing my or Roy his suggestion: we haven't made a concrete preferable approach. Choose what suits you.